locale/en.po

```diff
@@ -6018,9 +6009,9 @@ msgstr ""
 
 msgid "TUTORIAL_MICROBE_STAGE_REPRODUCE_TEXT"
 msgstr ""
-"To reproduce, you must duplicate your organelles. This happens gradually as you survive, but you can speed things up by gathering [thrive:compound type=\"ammonia\"][/thrive:compound] and [thrive:compound type=\"phosphates\"][/thrive:compound].\n"
+"Life needs more molecules than just sugar. See which other kinds of clouds your microbe can collect!\n"
```

Comment:
We consistently use "compounds" word in the game to mean molecules + other stuff so I think for consistent messaging this should not use the word molecules.